test/system/src/test/java/io/pravega/test/system/ReadTxnWriteScaleWithFailoverTest.java

```diff
@@ -106,18 +106,18 @@ public void setup() {
                 "ReadTxnWriteScaleWithFailoverTest-main");
         controllerExecutorService = ExecutorServiceHelpers.newScheduledThreadPool(2,
                 "ReadTxnWriteScaleWithFailoverTest-controller");
+        ClientConfig clientConfig = Utils.getClientConfig(controllerURIDirect);
         //get Controller Uri
         controller = new ControllerImpl(ControllerImplConfig.builder()
-                                    .clientConfig( ClientConfig.builder().controllerURI(controllerURIDirect).build())
+                                    .clientConfig(clientConfig)
                                     .maxBackoffMillis(5000).build(),
```

Comment:
This value repeats in several places. Can we move it as a constant to some file?

Reply:
Didn't get this..??

Reply:
I meant the number `5000`. It repeats everywhere, so it might make sense to use a descriptive constant to represent it, in case we need to change this later.

Reply:
There are a lot of such constants all over system tests and that need refactor. Can be done as a separate refactoring exercise rather than include in this PR which is not related.